e2e/Pages/ChatComponent.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from '@playwright/test';
+import { Locator, Page } from 'playwright';
+
+import EnglishTranslations from '../../interfaces/IBF-dashboard/src/assets/i18n/en.json';
+import DashboardPage from './DashboardPage';
+
+const chatDialogueContentWelcomeNoTrigger =
+  EnglishTranslations['chat-component'].floods['no-event-no-trigger'].welcome;
+const chatDialogueWarnLabel =
+  EnglishTranslations['chat-component'].common['warn-label'].message;
+
+class ChatComponent extends DashboardPage {
+  readonly page: Page;
+  readonly chatDialogue: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.chatDialogue = this.page.getByTestId('dialogue-turn-content');
+  }
+
+  async chatColumnIsVisibleForNoTriggerState({
+    name,
+    surname,
+  }: {
+    name: string;
+    surname: string;
+  }) {
+    // String cleaning to remove <strong> tags and replace placeholders with actual values
+    const cleanedString = chatDialogueWarnLabel.replace(/<\/?strong>/g, '');
+    const optionsDate: Intl.DateTimeFormatOptions = {
+      weekday: 'long',
+      day: '2-digit',
+      month: 'long',
+    };
+    const optionsTime: Intl.DateTimeFormatOptions = {
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: false,
+    };
+    const date = new Date();
+    const formattedDate = date
```

Comment:
This can be made convenient to implement and easier to understand if we use [date-fns](https://date-fns.org/v4.1.0/docs/lightFormat).

Reply:
Changed into date-fns format